axelor-base/src/main/java/com/axelor/apps/base/web/LanguageServiceImpl.java

```diff
@@ -0,0 +1,90 @@
+package com.axelor.apps.base.web;
+
+import com.axelor.apps.base.db.Language;
+import com.axelor.apps.base.db.repo.LanguageRepository;
+import com.axelor.db.Query;
+import com.axelor.inject.Beans;
+import com.axelor.meta.db.MetaSelect;
+import com.axelor.meta.db.MetaSelectItem;
+import com.axelor.meta.db.repo.MetaSelectItemRepository;
+import com.axelor.meta.db.repo.MetaSelectRepository;
+import com.google.inject.Inject;
+import com.google.inject.persist.Transactional;
+import java.util.List;
+
+public class LanguageServiceImpl implements LanguageService {
+  protected MetaSelectRepository metaSelectRepository;
+  protected MetaSelectItemRepository metaSelectItemRepository;
+  protected LanguageRepository languageRepository;
+
+  @Transactional
+  @Override
+  public void convertToLowercase(Language language) {
+    Language languageObj = Beans.get(LanguageRepository.class).find(language.getId());
+    languageObj.setCode(languageObj.getCode().toLowerCase());
+    languageRepository.save(languageObj);
+  }
```

Comment:
the constructor should always be before all the methods
But also this method is never used, delete it